src/theme/custom.css

```
@@ -77,3 +77,9 @@ th {
 .segment-wrapper {
   padding: 8px 16px;
 }
+@media (min-width:961px) {
+  .modal-wrapper {
+    --width: 90%;
+    --height: 80%;
```

I think the `height` can also be set to `90%`:

```suggestion
    --height: 90%;
```

---

Great idea